assets/component-product-model.css

```diff
@@ -3,6 +3,7 @@
   color: rgb(var(--color-foreground));
   margin: 1rem auto;
   box-shadow: none;
+  display: flex;
```

Comment:
It still seems to hide it properly with `display: none !important` applied to it from the `hidden` class. From here: 
![](https://screenshot.click/21-09-g7hml-kwjgi.png)

So maybe we just don't need that other display none 🤔

Reply:
That JS solution is only applied on Design Mode. The issue I mentioned above, related to the CSS property, is applied both inside and outside the Editor.

**Before / After (outside the Theme Editor)**
<img src="https://user-images.githubusercontent.com/37168033/134528332-a45cae11-ce12-4f10-9dfe-c72bcd8d5656.png" width="280"> <img src="https://user-images.githubusercontent.com/37168033/134528233-38539ddf-2e59-4f11-829b-e795106ef7c5.png" width="296">

Reply:
I can see needing the class `button` because we have a modifier class right after 🤔 `button--full-width`

Good catch on the flex. I am reverting that and will try to align it on the `ipad` breakdown differently

Reply:
Ah gotcha, I was focusing on the Editor 🤦  but I see it on the "live" version 👍  good catch.

Reply:
Oh, I wasn't suggesting to remove the `button` class from the element. Just the specificity on the CSS selector:

<img src="https://user-images.githubusercontent.com/37168033/134530532-2d01eb0b-c992-4dec-abf1-a5437a8aab8a.png" width="300">